package.json

```diff
@@ -34,7 +34,8 @@
     "react-scroll-sync": "^0.9.0",
     "remark-directive": "^2.0.1",
     "sass": "^1.35.1",
-    "signature_pad": "^3.0.0-beta.3"
+    "signature_pad": "^3.0.0-beta.3",
+    "uuid": "^9.0.0"
```

Comment:
There were some issues with `uuid`, downgrading might help. Best is to test it with the website and see if the tests go through.

Reply:
Thanks for the heads up, just downgraded the version, also for consistency among projects.